nodejs/week1/avg.js

```diff
@@ -0,0 +1,39 @@
+console.log("Hi I'm avg.js");
+
+const args = process.argv.slice(2);
+
+
+function checkIfNumber(arg) {
+   return !isNaN(parseFloat(arg)) && isFinite(arg);
+};
+
+function getAVG(args) {
+    const sum = args.reduce((acc, arg) => acc + parseFloat(arg), 0);
+    return sum / args.length;
+}
+
+function main(args) {
+    
+
+    if (args.length === 0) {
+        console.log("No arguments provided, please provide at least one number");
+        return;
+    } 
+    
+    if (!args.every(checkIfNumber)) {
+        console.log("Please provide only numbers");
+        return;
+    } 
+    
+    const average = getAVG(args);
+    console.log(`The average of ${args.join(", ")} is ${average}`);
+    return average;
+    } 
+
+    
+
+
+
+console.log(main(args));
```

Comment:
What is the motivation for this `console.log`? You already do all the printing in the `main()` function itself, this line of code will either output an unncesessary `undefined` or a duplicate value for average, which was already printed. You can remove this.